<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSettings</class>
 <widget class="QDialog" name="AudioSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="intercom.qrc">
    <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
  </property>
  <widget class="QGroupBox" name="gbAudioSampling">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Sampling</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbChannel">
       <property name="text">
        <string>Channeling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbChannels"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbBitPerSec">
       <property name="text">
        <string>Bit Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBitsRate"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbSamplesPerSec">
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSampleRate"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbTimeout">
       <property name="text">
        <string>Timeout (milliseconds)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="etTimeout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAudioTimeout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Additional Options</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="lbAudioCodec">
       <property name="text">
        <string>Audio Codec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAudioCodec"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>239</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="bnOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtering</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lbUpperThreshold">
       <property name="text">
        <string>Upper Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbUpperThreshold"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="lbLowerThreshold">
       <property name="text">
        <string>Lower Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLowerThreshold"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="intercom.qrc"/>
 </resources>
 <connections/>
</ui>
